external-crates/move/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -21,6 +21,7 @@ use crate::{
         StructName, UnaryOp, UnaryOp_,
     },
     shared::{unique_map::UniqueMap, *},
+    typing::core::is_public_for_testing,
```

Comment:
Could we add a visibility modifier `Visibility::PublicForTesting(Loc)` earlier to avoid needing to re-call this function?

Reply:
I think I can just override this let me fix it